specification/kubernetesconfiguration/resource-manager/readme.md

```diff
@@ -28,11 +28,24 @@ These are the global settings for the KubernetesConfiguration.
 title: SourceControlConfigurationClient
 description: KubernetesConfiguration Client
 openapi-type: arm
-tag: package-2022-03
+tag: package-preview-2022-04
 ```
```

Comment:
Lets use package-2022-03 as the default api version

Reply:
done.